unit tests fail on master

### Bug Description

unit tests fail

### To Reproduce

tox -e unit

### Environment

-

### Relevant log output

```shell
=============================================================================================================== FAILURES ===============================================================================================================
______________________________________________________________________________ TestKubernetesComputeResourcesPatch.test_invalid_config_emits_custom_event ______________________________________________________________________________
Traceback (most recent call last):
  File "/home/maksim/git/observability-libs/tests/unit/test_kubernetes_compute_resources.py", line 35, in setUp
    self.harness = Harness(
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 175, in __init__
    config_ = self._get_config(config)
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 466, in _get_config
    assert isinstance(charm_config, str)  # type guard
AssertionError
________________________________________________________________________ TestKubernetesComputeResourcesPatch.test_listener_is_attached_for_config_changed_event ________________________________________________________________________
Traceback (most recent call last):
  File "/home/maksim/git/observability-libs/tests/unit/test_kubernetes_compute_resources.py", line 35, in setUp
    self.harness = Harness(
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 175, in __init__
    config_ = self._get_config(config)
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 466, in _get_config
    assert isinstance(charm_config, str)  # type guard
AssertionError
__________________________________________________________________________ TestKubernetesComputeResourcesPatch.test_patch_is_applied_during_startup_sequence ___________________________________________________________________________
Traceback (most recent call last):
  File "/home/maksim/git/observability-libs/tests/unit/test_kubernetes_compute_resources.py", line 35, in setUp
    self.harness = Harness(
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 175, in __init__
    config_ = self._get_config(config)
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 466, in _get_config
    assert isinstance(charm_config, str)  # type guard
AssertionError
______________________________________________________________________ TestKubernetesComputeResourcesPatch.test_patch_is_applied_regardless_of_leadership_status _______________________________________________________________________
Traceback (most recent call last):
  File "/home/maksim/git/observability-libs/tests/unit/test_kubernetes_compute_resources.py", line 35, in setUp
    self.harness = Harness(
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 175, in __init__
    config_ = self._get_config(config)
  File "/home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/ops/testing.py", line 466, in _get_config
    assert isinstance(charm_config, str)  # type guard
AssertionError
=========================================================================================================== warnings summary ===========================================================================================================
.tox/unit/lib/python3.10/site-packages/_pytest/config/__init__.py:1253
  /home/maksim/git/observability-libs/.tox/unit/lib/python3.10/site-packages/_pytest/config/__init__.py:1253: PytestConfigWarning: Unknown config option: asyncio_mode
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================================================================================================= short test summary info ========================================================================================================
FAILED tests/unit/test_kubernetes_compute_resources.py::TestKubernetesComputeResourcesPatch::test_invalid_config_emits_custom_event - AssertionError
FAILED tests/unit/test_kubernetes_compute_resources.py::TestKubernetesComputeResourcesPatch::test_listener_is_attached_for_config_changed_event - AssertionError
FAILED tests/unit/test_kubernetes_compute_resources.py::TestKubernetesComputeResourcesPatch::test_patch_is_applied_during_startup_sequence - AssertionError
FAILED tests/unit/test_kubernetes_compute_resources.py::TestKubernetesComputeResourcesPatch::test_patch_is_applied_regardless_of_leadership_status - AssertionError
```


### Additional context

_No response_

--- a/tests/unit/helpers.py
+++ b/tests/unit/helpers.py
@@ -1,3 +1,6 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
 from pathlib import Path
 
 PROJECT_DIR = Path(__file__).resolve().parent.parent.parent

--- a/tests/unit/helpers.py
+++ b/tests/unit/helpers.py
@@ -1,0 +1,4 @@
+from pathlib import Path
+
+PROJECT_DIR = Path(__file__).resolve().parent.parent.parent
+UNITTEST_DIR = Path(__file__).resolve().parent

--- a/tests/unit/test_kubernetes_compute_resources.py
+++ b/tests/unit/test_kubernetes_compute_resources.py
@@ -5,7 +5,6 @@ from unittest import mock
 from unittest.mock import Mock
 
 import yaml
-
 from charms.observability_libs.v0.kubernetes_compute_resources_patch import (
     KubernetesComputeResourcesPatch,
     adjust_resource_requirements,

--- a/tests/unit/test_kubernetes_compute_resources.py
+++ b/tests/unit/test_kubernetes_compute_resources.py
@@ -4,6 +4,8 @@ import unittest
 from unittest import mock
 from unittest.mock import Mock
 
+import yaml
+
 from charms.observability_libs.v0.kubernetes_compute_resources_patch import (
     KubernetesComputeResourcesPatch,
     adjust_resource_requirements,
@@ -12,6 +14,8 @@ from charms.observability_libs.v0.kubernetes_compute_resources_patch import (
 )
 from ops.charm import CharmBase
 from ops.testing import Harness
+
+from tests.unit.helpers import PROJECT_DIR
 
 CL_PATH = "charms.observability_libs.v0.kubernetes_compute_resources_patch.KubernetesComputeResourcesPatch"
 
@@ -32,8 +36,10 @@ class TestKubernetesComputeResourcesPatch(unittest.TestCase):
             self.patch_failed_counter += 1
 
     def setUp(self) -> None:
+        with open(PROJECT_DIR / "config.yaml") as config_file:
+            config = yaml.safe_load(config_file)
         self.harness = Harness(
-            self._TestCharm, meta=open("metadata.yaml"), config=open("config.yaml")
+            self._TestCharm, meta=open(PROJECT_DIR / "metadata.yaml"), config=str(config)
         )
 
     @mock.patch("lightkube.core.client.GenericSyncClient", Mock)

